src/MeshKernelNET/Api/IMeshKernelApi.cs

```diff
@@ -186,6 +186,37 @@ int CurvilinearComputeTransfiniteFromTriangle(int meshKernelId,
         /// <returns>Error code</returns>
         int CurvilinearConvertToMesh2D(int meshKernelId);
 
+
+        /// <summary>
+        /// Delete the interior part of a curvilinear grid

```

Comment:
suggestion to rephrase: ... interior part of a rectangle in the curvilinear grid